From e74bb5a7ef87e697571181280e6108b89104c02d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 25 Jan 2023 20:40:56 -0800
Subject: [PATCH] include <cstdint>

gcc 13 libstdc++ moved some includes around and as a result <cstdint>
is no longer transitively included [1]. Explicitly include it for uintXX_t.

Fixes

../git/src/sw/redis++/utils.h:187:1: error: 'uint16_t' does not name a type
  187 | uint16_t crc16(const char *buf, int len);
      | ^~~~~~~~

[1] https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes

Upstream-Status: Submitted [https://github.com/sewenew/redis-plus-plus/pull/449]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/sw/redis++/utils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/sw/redis++/utils.h b/src/sw/redis++/utils.h
index f77f796..ff38f2d 100644
--- a/src/sw/redis++/utils.h
+++ b/src/sw/redis++/utils.h
@@ -17,6 +17,7 @@
 #ifndef SEWENEW_REDISPLUSPLUS_UTILS_H
 #define SEWENEW_REDISPLUSPLUS_UTILS_H
 
+#include <cstdint>
 #include <cstring>
 #include <string>
 #include <type_traits>
-- 
2.39.1

